provers/blevm/script/src/bin/tools.rs

```diff
@@ -0,0 +1,34 @@
+use clap::Parser;
+use rsp_client_executor::io::ClientExecutorInput;
+use std::{error::Error, fs};
+
+// The arguments for the command.
+#[derive(Parser, Debug)]
+#[clap(author, version, about, long_about = None)]
+struct Args {
+    #[clap(long)]
+    cmd: String,
+
+    #[clap(long)]
+    input: String,
+
+    #[clap(long)]
+    output: String,
+}
+
+#[tokio::main]
+async fn main() -> Result<(), Box<dyn Error>> {
+    // Load env variables.
+    dotenv::dotenv().ok();
+
+    // Parse the command line arguments.
+    let args = Args::parse();
+
+    if args.cmd == "dump-block" {
+        let mut cache_file = std::fs::File::open(args.input)?;
+        let client_input: ClientExecutorInput = bincode::deserialize_from(&mut cache_file)?;
+        let block_bytes = bincode::serialize(&client_input.current_block)?;
```

Comment:
[optional] maybe include an example usage of this command in comments or README?

Reply:
Sure. This is required because we need to submit the `ClientExecutorInput.current_block` as a blob to the da layer. I believe this will not be required in the demo as the rollup will writing full blocks instead of just the execution trace to the da layer.

Reply:
For reference, this is being read here: https://github.com/celestiaorg/celestia-zkevm-ibc-demo/blob/135fc354a2659f91fd3551694f5d1e37f6aa6655/provers/blevm/blevm-prover/src/lib.rs#L122-L130

Reply:
Will make an issue and do this later.